docs/MixPanel-batchsource.md

```diff
@@ -0,0 +1,22 @@
+# MixPanel batch source
+
+Description
+-----------
+This plugin used to query MixPanel events.
```

Comment:
This plugin used to fetch MixPanel events.

Reply:
done